pvlib/clearsky.py

```diff
@@ -195,22 +195,12 @@ def lookup_linke_turbidity(time, latitude, longitude, filepath=None,
         pvlib_path = os.path.dirname(os.path.abspath(__file__))
         filepath = os.path.join(pvlib_path, 'data', 'LinkeTurbidities.h5')
 
-    latitude_index = (
-        np.around(_linearly_scale(latitude, 90, -90, 0, 2160))
-        .astype(np.int64))
-    longitude_index = (
-        np.around(_linearly_scale(longitude, -180, 180, 0, 4320))
-        .astype(np.int64))
+    latitude_index = _degrees_to_index(latitude, degree_type='latitude')
+    longitude_index = _degrees_to_index(longitude, degree_type='longitude')
 
     lt_h5_file = tables.open_file(filepath)
-    try:
-        lts = lt_h5_file.root.LinkeTurbidity[latitude_index,
-                                             longitude_index, :]
-    except IndexError:
-        raise IndexError('Latitude should be between 90 and -90, '
-                         'longitude between -180 and 180.')
-    finally:
-        lt_h5_file.close()
+    lts = lt_h5_file.root.LinkeTurbidity[latitude_index, longitude_index, :]
```

Comment:
while we're at it, let's use a context manager to automatically open/close the file:

```python
with tables.open_file(filepath) as lt_h5_file:
    lts = lt_h5_file.root.LinkeTurbidity[latitude_index, longitude_index, :]
```

Reply:
These are all great ideas, and I've integrated them all with my latest commits.